### PR TITLE
Disallow factors for covariates in linear regression

### PR DESCRIPTION
**Pull request by vandenman**
Following the discussion in https://github.com/jasp-stats/INTERNAL-jasp/issues/1155

### CODE REVIEW

**Review by JohnnyDoorn — APPROVED** [review]

File: Resources/Regression/qml/RegressionLinear.qml [hunk 1]
```diff
@@ -38,7 +38,7 @@ Form
 				{ label: qsTr("Stepwise"),	value: "stepwise"}
 			]			
 		}
-		AssignedVariablesList { name: "covariates";	title: qsTr("Covariates");			suggestedColumns: ["scale"]								}
+		AssignedVariablesList { name: "covariates";	title: qsTr("Covariates");			allowedColumns: ["scale"]								}
```
Comment (TimKDJ) [inline Resources/Regression/qml/RegressionLinear.qml:41]: Should this also be done for the Bayesian one?

**Comment by JohnnyDoorn** [comment]
I think the Bayesian regression already disallows factors.

File: Resources/Regression/qml/RegressionLinear.qml (hunk 1 above)
Comment (JohnnyDoorn) [reply Resources/Regression/qml/RegressionLinear.qml:41]: I think the Bayesian regression already disallows factors.

**Comment by vandenman** [comment]
it is already disallowed:

https://github.com/jasp-stats/jasp-desktop/blob/7c2b238ff3e0ae6d6d7d2fcef6ed28e11bf14c1a/Resources/Regression/qml/RegressionLinearBayesian.qml#L29-L32

File: Resources/Regression/qml/RegressionLinear.qml (hunk 1 above)
Comment (TimKDJ) [reply Resources/Regression/qml/RegressionLinear.qml:41]: Ok, perfect

**Pull request merged**